src/App.tsx

```diff
@@ -1,15 +1,23 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [search, setSearch] = useState('');
+
+  const visibleMovies = moviesFromServer.filter(movie => {
+    const text = movie.title.toLowerCase() + movie.description.toLowerCase();
+    const searchPhrase = search.toLowerCase();
```

Comment:
![image](https://user-images.githubusercontent.com/102293862/198367861-7cd79125-cc23-4adc-a9a2-8507d923e317.png)
With this implementation I can search for the last word of title + the first order of description and it still works. Figure out how to fix this bug.